cmd/lncli/cmd_import_mission_control.go

```diff
@@ -66,7 +66,8 @@ func importMissionControl(ctx *cli.Context) error {
 		return fmt.Errorf("please provide amount in msat: %w", err)
 	}
 
-	if amt <= 0 {
+	// Allow 0 failure amount
```

Comment:
nit: comments should always be full sentences with punctuation. Also, do we want to update the error message? Since it's only true for success amounts now?